mailpoet/lib/API/JSON/v1/AutomatedLatestContent.php

```diff
@@ -93,7 +93,7 @@ public function getTransformedPosts(array $data = []): SuccessResponse {
     $posts = $this->getPermittedPosts($this->ALC->getPosts(new BlockPostQuery([
       'args' => $data,
       // If the request is for Posts block then we are fetching data for a static block
```

Comment:
Should this comment be updated (or maybe removed)?

Reply:
It should. Thank you! c36d93493